rascal-textmate-core/src/main/rascal/lang/rascal/grammar/analyze/Symbols.rsc

```diff
@@ -112,16 +113,6 @@ private map[Symbol, Maybe[set[Symbol]]] followBySymbol(Grammar g, bool(Symbol) p
     return ret;
 }
 
-private set[Symbol] unmaybe(just(set[Symbol] \set))
```

Comment:
Move the following functions to a new utility module, `util::MaybeUtil`

Reply:
looks to me like it should always have been a ``set`? as the empy set is a fine representation of "nothing"?